scripts/plugins/firewall-port

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# Copyright (c) Citrix Systems 2017. All rights reserved.
+
+set -e
+
+# Using this script to open/clear port. 
+
+
+OP="$1"
+PORT="$2"
+CHAIN="RH-Firewall-2-INPUT"
+RULE="-p tcp -m conntrack --ctstate NEW -m tcp --dport $PORT -j ACCEPT"
+
+case "${OP}" in
+    open)
+        if iptables -S | grep $CHAIN | grep -e "${RULE}" > /dev/null
+        then
+            echo $"port $2 is opened already"
+        else
+            if iptables -S | grep -e "${CAHIN}" > /dev/null
+            then
+                echo $"$CHAIN already exists"
+            else
+                iptables -N $CHAIN
+                iptables -I INPUT -j $CHAIN
+            fi
+            iptables -I $CHAIN $RULE
+            service iptables save
+        fi
+        ;;
+    close)
+        iptables -D $CHAIN $RULE
+        service iptables save
+        ;;
+    *)
+        echo $"Usage: $0 {open|clear}" 1>&2
```

Comment:
hi @rdobson . I have updated the code. @thomassa  do you have any suggestions for the new chain added methods?